packages/frontend/app/routes/_index.tsx

```diff
@@ -1,3 +1,30 @@
+import { version } from '../../../../package.json'
+
 export default function Index() {
-  return <></>
+  return (
+    <div className='pt-4 flex flex-col'>
+      <div
+        className='flex flex-col rounded-md bg-offwhite px-6 text-center'
+        style={{ minHeight: 'calc(100vh - 3rem)' }}
+      >
+        <div className='p-10 space-y-16'>
+          <h1 className='text-9xl pt-16 text-[#F37F64]'>Rafiki Admin</h1>
+          <div className='space-y-8'>
+            <p className='text-7xl'>Welcome!</p>
```

Comment:
Nit pick: can we swap those two? Have `Welcome` big and flashy and then `Rafiki Admin` below it? Maybe even 
```
Welcome!
to
Rafiki Admin
```
?